docs/pages/blog/lab-pickers-to-mui-x.md

```diff
@@ -0,0 +1,63 @@
+---
+title: The date pickers are now part of the Advanced components
+description: XXXX
+date: 2022-04-03T00:00:00.000Z
+authors: ['flaviendelangle']
+tags: ['MUI X', 'News']
+---
+
+We're excited to announce that, after more than 18 month in the lab, the date picker components are now part of the X packages.
+
+## What is MUI X ?
+
+Since its creation, MUI have always kept at heart to provide great components for free.
+However, some _Advanced components_ such as date pickers require several full-time developers and can not be sustainable relying only on open-source income.
+To be able to keep improving the library and add _Advanced components_ without compromise on code quality, we decided in 2020 to launch MUI X, a new set of components available under two licenses:
+
+- MUI X: the MIT licensed, free to use version
+- MUI X Pro: the commercially licensed version with the most advanced features
+
+The Advanced component launched by our team have been the Data Grid, with the `@mui/x-data-grid` and `@mui/x-data-grid-pro` packages.
+More than one year later, this new source of income has allowed our company to hire several new developers and to increase the amount of work achieved, even on the open-source team.
+
+## Why move the date pickers to MUI X
+
+When the date pickers components have moved to the lab, it was already clear that some of their features would eventually be part of the commercially licensed packages.
```

Comment:
Let's elaborate on why the decision was made, and why it matters to the reader/user.

```suggestion
Date picker UI elements appear simple enough on the surface, but they are surprisingly complicated to engineer.

When our date picker components were ready to leave our experimental lab, we had a choice to make: would they live in the MUI Core library, or should they become a part of MUI X?

Given the complex nature of these components, we decided that they would benefit most from the dedicated attention that they would receive from the X team.

And by offering an even more advanced component package through our commercial license, we can continue to grow our team to better serve your needs.
```